src/__internal__/validation-message/validation-message.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { mount } from "enzyme";
+import ValidationMessage from ".";
+import StyledValidationMessage from "./validation-message.style";
+import { assertStyleMatch } from "../../__spec_helper__/test-utils";
+import { baseTheme } from "../../style/themes";
+
+const render = (props) => mount(<ValidationMessage {...props} />);
+
+const validationWithStrings = [
+  ["error", "warning"],
+  [undefined, "warning"],
+];
+const validationWithBooleans = [
+  [true, "string"],
+  [undefined, true],
+];
+
+const computeId = (id) => (id === "warning" ? "warningText" : id);
+
+const addColorId = (arr) =>
+  arr.map((subArr) => [...subArr, computeId(subArr[0] || subArr[1])]);
+
+describe("ValidationMessage component", () => {
+  let wrapper;
+
+  it.each(addColorId(validationWithStrings))(
+    "applies the expected styling when passed error is `%s`, warning is `%s` and info is `%s`",
```

Comment:
This is quite confusing with the old validation `info` 
```suggestion
    "applies the expected styling when passed error is `%s`, warning is `%s` and message is `%s`",
```

Reply:
ah yeah I missed removing info from that thanks will get it sorted